core/src/main/java/io/temporal/samples/countinterceptor/SimpleCountWorkerInterceptor.java

```diff
@@ -34,4 +36,10 @@ public WorkflowInboundCallsInterceptor interceptWorkflow(WorkflowInboundCallsInt
   public ActivityInboundCallsInterceptor interceptActivity(ActivityInboundCallsInterceptor next) {
     return new SimpleCountActivityInboundCallsInterceptor(next);
   }
+
+  @Override
+  public NexusOperationInboundCallsInterceptor interceptNexusOperation(
```

Comment:
The "correct" solution is these samples should be `SimpleCountWorkerInterceptor extends BaseWorkerInterceptor `